.github/workflows/ci_suite.yml

```diff
@@ -85,4 +85,4 @@ jobs:
         uses: re-actors/alls-green@release/v1
         with:
           jobs: ${{ toJSON(needs) }}
-          allowed-skips: compare_screenshots
+          allowed-skips: compare_screenshots,run_alternate_tests
```

Comment:
this seems wrong to me

Reply:
otherwise completion gate will always 'fail' if alternative tests were skipped

Reply:
well.. the alternate tests were required before. Right now they are empty because there is no beta version and tests are on 516.1659 (latest)